cfme/web_ui/__init__.py

```diff
@@ -2342,7 +2342,8 @@ def a_cond(self):
             return "contains(normalize-space(@title), {name})"\
                 .format(name=quoteattr('Name: {}'.format(self._name)))
         else:
-            return "@title={name} or @data-original-title={name}".format(name=quoteattr(self._name))
+            return "@title={name} or @data-original-title={name}"\
```

Comment:
I like what you are suggesting better, but I wanted to stay consistent with the above.